src/main/java/uk/gov/hmcts/reform/juddata/validators/AppointmentValidator.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.juddata.validators;
+
+import org.apache.commons.lang3.StringUtils;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.util.Set;
+
+public class AppointmentValidator implements ConstraintValidator<Appointment, String> {
+
+    private static final String VALUE1 = "CRTS TRIB - RS Admin User";
+    private static final String VALUE2 = "MAGS - AC Admin User";
```

Comment:
Can you please move these in constant class

Reply:
These constants are not reusable and strongly tied to existing class.

According to Joshua Bloch in his book Effective Java (Item 19 in 2nd Ed):

If the constants are strongly tied to an existing class or interface, you should add them to the class or interface.